.experimenter.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+address-autofill-feature:
+  description: "Feature description\n"
```

Comment:
@adudenamedruby is this normal this is `"Feature description\n"`only? Here and under the variables.description as well

Reply:
Good catch I pushed a description